nitro-shim/tools/utils/go.mod

```diff
@@ -0,0 +1,3 @@
+module nitro-shim/utils
+
+go 1.17
```

Comment:
bump 1.18

Reply:
Should be addressed in https://github.com/brave-intl/bat-go/commit/4e928e5b0d36c6f5e6b195ea73e79b07a86690a1.